infrastructure/rp-events/template.yaml

```diff
@@ -140,6 +140,32 @@ Resources:
             Resource:
               - '{{resolve:ssm:DatabaseKmsKeyArn}}'
 
+  UserJourneyIdCacheTable:
```

Comment:
can we export table arn as

1. ssm param
2. cloudformation export

as we need to access it in event processing

Reply:
exported just as ssm param as discussed on call with Richa and Zayd